src/js/index.js

```diff
@@ -0,0 +1,43 @@
+const numberCount = 0;
+const total = document.getElementById('total');
+const operatorCheck = true;
+
+
+function add(char) {
+    if (isNaN(char)) {
+        if (this.numberCount == 0 && this.total.innerText == '0') {
+            alert('숫자를 먼저 입력한 후 연산자를 입력해주세요!');
+        } else if (this.operatorCheck) {
+            this.total.innerText += char;
+            this.operatorCheck = false;
+            this.numberCount = 0;
+        } else {
+            alert('연산자는 하나만 입력할 수 있습니다.');
+        }
+
+    } else if (this.total.innerText == '0') {
+        console.log('0일때')
+        this.total.innerText = char;
+        this.numberCount = 1;
+        this.operatorCheck = true;
+    } else if (this.numberCount < 3) {
+        console.log('3미만')
+        this.total.innerText += char;
+        this.numberCount += 1;
+    } else if (this.numberCount >= 3) {
+        alert('숫자는 세 자리까지만 입력 가능합니다!');
+    }
+}
+
+function reset() {
+    this.total.innerText = '0';
+    this.numberCount = 0;
+    this.operatorCheck = true;
```

Comment:
전역변수는 바로 접근할 수 있기 때문에 지금 코드에서 `this`는 전부 생략해도 될 것 같습니다~
```suggestion
    total.innerText = '0';
    numberCount = 0;
    operatorCheck = true;
```